apps/web/app/clusters/metrics-board.tsx

```diff
@@ -0,0 +1,228 @@
+                    </div>
+                ) : (
+                    <Button className="flex gap-2" onClick={() => setShouldEdit(true)}><PencilIcon className="h-5 w-5"/> Edit</Button>
+                )}
+            </div>
+
+            <hr className="border-slate-500" />
+
+            <div className={`flex flex-wrap justify-center gap-4`}>
+                {metrics.map((item: DashboardItem, i) => (
+                    <Tile key={i} className="rounded-md w-full min-h-40 max-w-[416px] p-3 flex flex-row gap-2">
+                        <div className="flex justify-between w-full">
+                            <div className="flex flex-col gap-1">
+                                <h3>{item.title}</h3>
+                                <p>{item.description}</p>
+                                {item.seeAll && <p className="hover:underline"><a href={item.seeAllLink}>See all</a></p>}
+                            </div>
+                                
+                            <span>{getChart(item)}</span>
+                        </div>
+                        {shouldEdit && (
+                            <button onClick={() => removeMetric(item.id)}  className="text-red-500 hover:text-red-600 h-fit">
+                                <CrossIcon /> 
+                            </button>
+                        )}
+                    </Tile>
+                ))}
+                {shouldEdit && 
+                    <Tile className="rounded-md w-full min-h-40 max-w-[416px] p-3 flex flex-col gap-6 border border-transparent hover:border-neutral-200 transition-colors duration-150">
+                        <div className="flex flex-col gap-2">
+                            <div className="flex flex-col gap-1">
+                                <h3>Add new metric</h3>
+                            </div>
+                            <p>What metric do you want to add?</p>
+                        </div>
+
+                        <form className="flex flex-row gap-2" onSubmit={handleSubmit(addMetric)}>
+                            <Controller
+                                name="metric"
+                                control={control}
+                                render={({ field }) => (
+                                    <select {...field} className="w-fit rounded-md bg-neutral-800 border border-neutral-200 py-2 px-4">
+                                        {metricTypes.map((type) => (
+                                            <option key={type.typeId} value={type.title}>{type.title}</option>
+                                        ))}
+                                    </select>
+                                )}
+                            />
+                            <Button type="submit" className="max-h-9">Add</Button>
+                        </form>
+                    </Tile>
+                }
+            </div>
+        </div>
+    )
+}
+
+export default MetricsBoardProps
```

Comment:
```suggestion
export default MetricsBoard
```

Reply:
I would also suggest to use named exports as it generally becomes easier to work with as the codebase grows.